tls/s2n_client_extensions.c

```diff
@@ -137,7 +137,7 @@ int s2n_client_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *
         GUARD(s2n_extensions_client_supported_versions_send(conn, out));
     }
 
-    if (conn->actual_protocol_version == S2N_TLS12) {
+    if (conn->actual_protocol_version >= S2N_TLS12) {
```

Comment:
/nit I would prefer being more specific instead of `>=` because it may not hold true after TLS13?

Reply:
Even versions after TLS 1.3 would keep the old extensions around for backwards compatibility, and all future versions behaving like the previous version unless otherwise specified seems like the correct behavior.